### PR TITLE
[IMP] project: exclude sub-tasks from count

### PR DESCRIPTION
**Pull request by bastvdn**
Task-3846136

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/project/models/project_project.py [hunk 1]
```diff
@@ -26,9 +26,13 @@ def _compute_task_count(self):
         project_and_state_counts = self.env['project.task'].with_context(
             active_test=any(project.active for project in self)
         )._read_group(
-            [('project_id', 'in', self.ids)],
-            ['project_id', 'state'],
-            ['__count'],
+            domain=[
```
Comment (auon-odoo) [inline addons/project/models/project_project.py:29]: Me personally, I prefer it with keywords, as I find it more readable. But @xavierbol, didn't you tell me it was more efficient without?

**Review by auon-odoo — APPROVED** [review]
LGTM :zap:

File: addons/project/models/project_project.py (hunk 1 above)
Comment (xavierbol) [reply addons/project/models/project_project.py:29]: @auon-odoo 
TBH, that's what I heard but I don't know the gain and I don't know if it is true or not.

And so, I would say you can do what you prefer. 🙂

**Comment by xavierbol** [comment]
robodoo r+

**Pull request closed without merging**